frontend/src/container/PublicProfile.jsx

```diff
@@ -384,7 +384,7 @@ export class PublicProfile extends React.Component {
     const { props, state } = this
 
     const userId = state.displayedUser ? state.displayedUser.userId : props.match.params.userid
-    const isPublicProfileEditable = this.isPublicProfileEditable(props.user, userId, PROFILE)
+    const isPublicProfileEditable = this.isPublicProfileEditable(props.user, userId, PROFILE) && state.dataSchemaObject.properties
```

Comment:
this is valid even if the custom properties are deactivated? because this const is used for the avatar and the cover too.

Reply:
see if https://github.com/tracim/tracim/issues/4102 is not enough